src/core.js

```diff
@@ -1670,7 +1670,7 @@ define([
 
         this.$f.find('.fd-props-toolbar').html(this.getMugToolbar(mug));
         for (var i = 0; i < sections.length; i++) {
-            var section = sections[i];
+            var section = _.clone(sections[i]);
```

Comment:
Why is this needed?

Reply:
because `section` is mutated below on line [1676](https://github.com/dimagi/Vellum/pull/924/files#diff-7eb52b366866677666470e019283c8eaR1676)